.vscode/settings.json

```diff
@@ -43,5 +43,6 @@
         "**/*.d.ts": true,
         "assets": true
     },
+    "editor.tabSize": 4,
```

Comment:
nope :)

Reply:
Maybe this should not be part of this PR. Several files seem to use four spaces. Does it make sense?

Reply:
Well actually you are right this is ok just to have everything aligned